PermissionsEx/permissions.yml

```diff
@@ -2302,6 +2302,25 @@ groups:
         - essentials.skull
         - essentials.skull.*
         - essentials.gamemode.survival
+  doctordoom:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
+        - ch.alias.hoveron
+        - ch.alias.hoveroff
+        - essentials.tp
+        - essentials.tp.*
+        - essentials.gamemode
+        - essentials.gamemode.creative
+        - essentials.skull
+        - essentials.skull.*
```

Comment:
https://images.search.yahoo.com/yhs/search;_ylt=AwrTcYRlNn1Yh8QABhE2nIlQ;_ylu=X3oDMTBsZ29xY3ZzBHNlYwNzZWFyY2gEc2xrA2J1dHRvbg--;_ylc=X1MDMTM1MTE5NTcwMgRfcgMyBGFjdG4DY2xrBGJjawNic2c0NXI5YjQxaW1nJTI2YiUzRDQlMjZkJTNET2tYb0hVZHBZRUlrQ0NxeWxOYmg5Xy5RMEliekwxUWxnLjBxdVEtLSUyNnMlM0RpaCUyNmklM0RCOG5lYU00WXU5MHNmN3AueFlkYQRjc3JjcHZpZANmR29BYVRJd05pNi5RSVhhVmtESzBBQm1Oamd1TVFBQUFBQ1hSWklyBGZyA3locy1vbXItMDAxBGZyMgNzYS1ncARncHJpZANPVmx0OE1YTFNuR0trNkZQcTViX0hBBG10ZXN0aWQDbnVsbARuX3N1Z2cDMARvcmlnaW4DaW1hZ2VzLnNlYXJjaC55YWhvby5jb20EcG9zAzAEcHFzdHIDBHBxc3RybAMEcXN0cmwDMzEEcXVlcnkDRG9jdG9yIERvb20gZHVwbGljYXRpbmcgaGltc2VsZgR0X3N0bXADMTQ4NDYwMTAxNAR2dGVzdGlkA251bGw-?gprid=OVlt8MXLSnGKk6FPq5b_HA&pvid=fGoAaTIwNi6.QIXaVkDK0ABmNjguMQAAAACXRZIr&p=Doctor+Doom+duplicating+himself&fr=yhs-omr-001&fr2=sb-top-images.search.yahoo.com&ei=UTF-8&n=60&x=wrt&type=86311436&hsimp=yhs-001&hspart=omr#id=8&iurl=http%3A%2F%2Fblog.mysanantonio.com%2Fgeekspeak%2Ffiles%2F2012%2F03%2Fdoctor_vs__doctor_by_docshaner-d36i641.jpg&action=click

He can create more of himself out of nothing